onnxruntime/core/providers/cpu/rnn/rnn_helpers.h

```diff
@@ -158,7 +160,64 @@ void ComputeGemm(const int M,
       M, N, K, alpha,
       &*A, lda,
       &*B, ldb, beta,
-      &*C, ldc, tp);
+      &*C, ldc, thread_pool);
+}
+
+struct PackedWeights {
+  BufferUniquePtr buffer_;
+  size_t weights_size_;
```

Comment:
nit: using IAllocatorUniquePtr and IAllocator::MakeUniquePtr would be slightly more consistent with the other unique_ptr allocations (we really shouldn't have two ways to do the equivalent thing though).